lib/nextflow-config/index.ts

```diff
@@ -144,7 +144,7 @@ export class NextflowConfigConstruct extends Construct {
         // this means CDK will handle deploying it to ECR for us
         const imageAsset = new DockerImageAsset(this, configName, {
           directory: join(__dirname, "thin-docker"),
-          platform: Platform.LINUX_AMD64,
+          platform: Platform.LINUX_ARM64,
```

Comment:
Similar to @andrewpatto's comment, I'd possibly make this a configurable option in the settings.ts if it's simple to do (i.e. the changes to docker images below are compatible with both AMD64 and ARM64?)